### PR TITLE
Add chDB integration  doc

### PR DESCRIPTION
**Pull request by nmreadelf**
Closes https://github.com/chdb-io/chdb/issues/15

----
:books: Documentation preview :books:: https://jupysql--857.org.readthedocs.build/en/857/

### CODE REVIEW

**Comment by neelasha23** [comment]
Docs build is failing @nmreadelf

**Comment by nmreadelf** [comment]
> Docs build is failing @nmreadelf

thx, i fixed

File: doc/integrations/chdb.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+%pip install jupysql chdb --quiet
+```
+
+```{code-cell} ipython3
+from chdb import dbapi
+conn = dbapi.connect()
+
+%load_ext sql
+%sql conn --alias chdb
+```
```
Comment (edublancas) [inline doc/integrations/chdb.md:97]: looks like this section is showing how to add a new connection, I think we can delete it. we already have documentation on how to manage connections

File: doc/integrations/chdb.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+%%sql
+SELECT
+    repo_name AS repo,
+    count() AS stars
+FROM remoteSecure('play.clickhouse.com', default, github_events, 'play', 'play')
+WHERE event_type = 'WatchEvent' AND created_at > today() -1
+GROUP BY repo_name
+ORDER BY stars DESC
+LIMIT 10
+```
```
Comment (edublancas) [inline doc/integrations/chdb.md:115]: we can delete this as well

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,7 @@
 * [Feature] Improved messages when loading configurations from `pyproject.toml` file.
 * [Feature] Add `--schema/-s` for `%sqlcmd` commands that support `--table/-t` and ensure `--table schema.table` works (#519)
 * [Feature] Show feedback when starting a new connection (#807)
+* [Feature] Add chDB integration doc
```
Comment (edublancas) [inline CHANGELOG.md:8]:
```suggestion
* [Doc] Add chDB integration tutorial
```

feel free to add your github username at the end

**Review by edublancas — CHANGES_REQUESTED** [review]
the documentation build is failing

you need to add your doc here: https://github.com/ploomber/jupysql/blob/master/doc/_toc.yml

then, you'll need to add chdb here:

https://github.com/ploomber/jupysql/blob/master/doc/environment.yml
https://github.com/ploomber/jupysql/blob/master/doc/environment.lock.yml

File: doc/integrations/chdb.md (hunk 2 above)
Comment (nmreadelf) [reply doc/integrations/chdb.md]: thx

**Comment by nmreadelf** [comment]
> the documentation build is failing
> 
> you need to add your doc here: https://github.com/ploomber/jupysql/blob/master/doc/_toc.yml
> 
> then, you'll need to add chdb here:
> 
> https://github.com/ploomber/jupysql/blob/master/doc/environment.yml https://github.com/ploomber/jupysql/blob/master/doc/environment.lock.yml

thx, fixed

**Comment by edublancas** [comment]
awesome, thanks! @nmreadelf feel free to come back to add new stuff to the tutorial. chDB is an exciting project!

**Pull request merged**